packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -307,7 +310,12 @@ const Updates = decorate([
                 </p>
                 {state.showPackagesList && (
                   <p>
-                    <Copiable tagName='pre'>{state.packagesList}</Copiable>
+                    <Copiable tagName='pre'>
+                      <p>
+                        - {_('xoaBuild')}: {state.xoaBuild}
+                      </p>
```

Comment:
This is unrelated to the packages, I think we should display in a similar fashion than `xen-orchestra` version.